build.gradle

```diff
@@ -155,12 +156,12 @@ uploadArchives {
 
                 developers {
                     developer {
-                        name 'Lars Bärtschi'
-                        email 'lars@twasi.net'
+                        name 'Christophe Carvalho Vilas-Boas'
+                        email 'christophe.carvalhovilasboas@gmail.com'
                     }
                     developer {
-                        name 'Merlin Westphal'
-                        email 'merlin@twasi.net'
+                        name 'TinaTiel'
+                        email 'contact@tinatiel.com'
```

Comment:
@Pjiesco Maybe you can add yourself in here too

Reply:
Yup will do